src/Loader.php

```diff
@@ -64,7 +64,7 @@ public function __construct() {
 		add_filter( 'woocommerce_settings_groups', array( __CLASS__, 'add_settings_group' ) );
 		add_filter( 'woocommerce_settings-wc_admin', array( __CLASS__, 'add_settings' ) );
 		add_action( 'admin_head', array( __CLASS__, 'remove_notices' ) );
-		add_action( 'admin_notices', array( __CLASS__, 'inject_before_notices' ) );
+		add_action( 'admin_notices', array( __CLASS__, 'inject_before_notices' ), -9999 );
```

Comment:
Ideally we would use something like `PHP_INT_MAX`, but this isn't supported in PHP 5.6. I'm not sure how low to go here.

Reply:
Did you mean `PHP_INT_MIN` here?  I think max is supported.  I'm guessing we could do `PHP_INT_MAX * -1` but that may throw errors on weird edge cases where a server had min/max configured differently.

I think `-9999` will cover most, if not all cases though.

Reply:
Yeah, I meant `PHP_INT_MIN` here, sorry.

Let's stick with `-9999` for now. If we get reports of something not working we can adjust later.